platform/lib/routers/surveyComponent.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Joi = require('joi');
+const express = require('express');
+const log = require('@lib/utils/log')('Survey Component Response');
+const {GoogleSpreadsheet} = require('google-spreadsheet');
+const Doc = new GoogleSpreadsheet(process.env.SURVEY_RESPONSE_SHEET_ID);
```

Comment:
Would it make sense to configure this in our environment specific config variables? https://github.com/ampproject/amp.dev/tree/future/platform/config/environments

Reply:
I am not sure - do you consider this information public? We wouldn't be exposing it elsewhere. I had assumed it would be configured on the Cloud settings, along with the private key and account information

Reply:
Makes sense. Let's configure this via the cloud settings.